uaclient/tests/test_cli_status.py

```diff
@@ -19,10 +19,23 @@
 RESPONSE_AVAILABLE_SERVICES = [
```

Comment:
Shouldn't we also have a test where we have a service that is not present on the contracts response ?

Reply:
Good point. Will remove one of those